src/renderer/components/editors-non-ideal-state.tsx

```diff
@@ -1,6 +1,5 @@
-import * as React from 'react';
-
 import { Button, NonIdealState } from '@blueprintjs/core';
+import * as React from 'react';
```

Comment:
Is it possible to have `React` always in the first row? with the highest priority

Reply:
I think that might require a separate plugin here: https://github.com/lydell/eslint-plugin-simple-import-sort

Reply:
We can do this by defining custom `pathGroups` and ignoring the pattern then - [docs](https://github.com/import-js/eslint-plugin-import/blob/5fe9aa44eb9d20b376d1b32a5d3384c46a01309e/docs/rules/order.md#pathgroupsexcludedimporttypes-array)
```js
{
  "import/order": [
    "error",
    {
      "pathGroups": [
        {
          "pattern": "react",
          "group": "builtin",
          "position": "before"
        }
      ],
      "pathGroupsExcludedImportTypes": ["react"]
    }
  ]
}
```

This is how the sorting order looks like then - 
```js
import * as React from 'react';

import { Button, NonIdealState } from '@blueprintjs/core';

import { EditorMosaic } from '../editor-mosaic';

export function renderNonIdealState(editorMosaic: EditorMosaic) {
```

Reply:
> I think that might require a separate plugin here: https://github.com/lydell/eslint-plugin-simple-import-sort

I thought of using this first until I read [this para](https://github.com/lydell/eslint-plugin-simple-import-sort#not-for-everyone) in its docs. (I thought it's better to have more control on how we sort the imports)
Quoting a para from its docs - 
> If you want more options, I recommend using the [import/order](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) rule (from [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import/)) instead. It has plenty of options, and the maintainers seem interested in expanding the feature where it makes sense.